server/src/test/java/io/crate/integrationtests/PgStatsTableTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import org.elasticsearch.test.ESIntegTestCase;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+
+@ESIntegTestCase.ClusterScope(numDataNodes = 1, numClientNodes = 0)
+public class PgStatsTableTest extends SQLTransportIntegrationTest {
+
+    @Test
+    public void test_select_from_pg_stats_when_most_common_vals_is_array_type_value() {
```

Comment:
maybe add this to https://github.com/crate/crate/blob/master/server/src/test/java/io/crate/integrationtests/AnalyzeITest.java

Otherwise I  think it won't be clear which test cases should go to what test class as both are closely related